system-x/services/splunk/src/main/java/software/tnb/splunk/account/SplunkAccount.java

```diff
@@ -0,0 +1,26 @@
+package software.tnb.splunk.account;
+
+import software.tnb.common.account.Account;
+
+public class SplunkAccount implements Account {
+
+    private String username = "admin";
+
+    private String password;
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
```

Comment:
change getters to `username()` and `password()` to follow the same style as in other accounts